app.js

```diff
@@ -16,13 +21,10 @@ db.once('open', function() {
   });
 });
 
+const xxx = db.collection('users')
```

Comment:
we should probably change this const to something more descriptive, but we can do that next PR